### DISCUSSION TITLE
Is GitHub Android version Open Source?

### DISCUSSION THREAD

**Discussion by ItsPiyusha** [EddieHubCommunity/support#6255] [Q&A] [answered]
### Description of feature request, idea

If yes, can I get the link to that? It will be very helpful for all of us to contribute even through our phone. So let's try to make it more user friendly, code friendly.

[a comment by an automated account is not shown]

**Comment by schmelto** [comment, marked as answer]
Do you mean the GitHub mobile app?
https://github.com/mobile

**Comment by ItsPiyusha** [comment]
yes
